docs/apache-airflow/core-concepts/executor/local.rst

```diff
@@ -47,3 +47,13 @@ Arguably, :class:`~airflow.executors.sequential_executor.SequentialExecutor` cou
 parallelism of just 1 worker, i.e. ``self.parallelism = 1``.
 This option could lead to the unification of the executor implementations, running
 locally, into just one :class:`~airflow.executors.local_executor.LocalExecutor` with multiple modes.
+
+.. note::
+
+   When multiple Schedulers are configured with ``executor = LocalExecutor`` in the ``[core]`` section of your ``airflow.cfg``, each scheduler will run a LocalExecutor. This means tasks would be processed in a distributed fashion across the machines running the Schedulers.
```

Comment:
Make a mind whether to capitalise scheduler or not, don’t mix.

Reply:
Done.

Reply:
You missed one

Reply:
@uranusjr done now